vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -490,6 +496,32 @@ class HomeActivityViewModel @AssistedInject constructor(
         }
     }
 
+    /**
+     * Stop ongoing voice broadcast if any.
+     */
+    private fun stopOngoingVoiceBroadcast() {
+        val session = activeSessionHolder.getSafeActiveSession() ?: return
```

Comment:
Do you think it is possible to encapsulate all this code in a dedicated `UseCase` such as `StopOngoingVoiceBroadcastUseCase`? We could add unit tests on it later.